numpy/lib/shape_base.py

```diff
@@ -269,7 +269,7 @@ def apply_along_axis(func1d, axis, arr, *args, **kwargs):
     """
     Apply a function to 1-D slices along the given axis.
 
-    Execute `func1d(a, *args)` where `func1d` operates on 1-D arrays and `a`
+    Execute `func1d(a, *args, **kwargs)` where `func1d` operates on 1-D arrays and `a`
```

Comment:
Line is too long, please keep lines <= 75 characters for docstrings.

Reply:
fixed now